lib/evss/response.rb

```diff
@@ -7,10 +7,21 @@ class Response < Common::Base
     include Common::Client::ServiceStatus
 
     attribute :status, Integer
+    attribute :body, Object
 
-    def initialize(status, response = nil)
-      self.status = status
-      super(response.body) if response
+    def initialize(response)
+      case response
+      when Faraday::Response
+        self.status = response.status
+        parse_body(response.body)
+      else
+        self.status = response[:status]
+        parse_body(response[:body]) if response[:body]
```

Comment:
This looks to be doing too much.  Might I recommend that the method signature look like this:
```ruby
def initialize(status, attributes)
```
and push the responsibility of teasing actual data out of the Faraday response down into the respective `EVSS::Response` inherited response class.  For example, `GiBillStatusResponse` would have:
```ruby
def initialize(status, response = nil)
  attributes = response.body['chapter33_education_info']
  super(status, attributes)
end
```
and `BeneficiaryResponse` would have
```ruby
def initialize(status, response = nil)
  attributes = response.body
  super(status, attributes)
end
```